cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -200,6 +211,12 @@ protected String getDefaultLogConfigFileName() {
     }
 
     public Deployment generateDeployment() {
+
+        if (!isDeployed()) {
+            log.warn("Topic and/or User Operators not declared: Entity Operator will not be deployed");
```

Comment:
Is this really a warning? Wouldn't it be just `INFO`? It isn't necessarily wrong to not use them.

Reply:
The warning is about saying "hey, you have declared the 'entityOperator` field in the spec but it's empty". Of course, you have the freedom to not use TO and UO but you should not declare an empty `entityOperator`: if you don't declare this empty field, no warning appears.